tools/generators/recipe-generator/index.spec.ts

```diff
@@ -0,0 +1,56 @@
+import { createTreeWithEmptyWorkspace } from '@nrwl/devkit/testing';
+import { Tree } from '@nrwl/devkit';
+import * as path from 'path';
+
+import generator from './index';
+import { RecipeGeneratorGeneratorSchema } from './schema';
+
+const recipesRoot = 'apps/recipes-react-components/src/recipes';
+const recipeName = 'Hello World';
+const recipePackageName = 'hello-world';
+
+describe('recipe-generator generator', () => {
+  let tree: Tree;
+  const options: RecipeGeneratorGeneratorSchema = { recipeName };
+
+  beforeEach(() => {
+    tree = createTreeWithEmptyWorkspace();
+  });
+
+  it('should generate boilerplate', async () => {
+    await generator(tree, options);
+
+    ['HelloWorld.stories.mdx', 'code-snippets/HelloWorld.tsx', 'code-snippets/index.ts', 'code-snippets', ''].forEach(
+      file => {
+        expect(tree.exists(path.join(recipesRoot, recipePackageName, file))).toBeTruthy();
+      },
+    );
```

Comment:
For some reason these don't fail locally ??. Tried inline snapshot and also not working :/

Reply:
I'm getting same failures on local machine as happen on CI.

Reply:
ok so I checked in more detail and the implementation will need to be tweaked. I'll  add comments later.

Reply:
thanks for taking a look!